### PR TITLE
Allow refreshing bookmark list

### PR DESCRIPTION
**Pull request by rasendubi**
Problem: bookmarks are only updated on restart. This is inconvenient
if you often save bookmarks from other applications. (Or from the same
one.)

Solution: allow the manual refresh of articles list.

### CODE REVIEW

File: src/components/ArticlesPage.js [hunk 1]
```diff
@@ -15,13 +16,35 @@ const styles = StyleSheet.create({
 });
 
 export default class ArticlesPage extends Component { // eslint-disable-line
+  constructor(props) {
+    super(props)
+
+    const ds = new ListView.DataSource({
+      rowHasChanged: (r1, r2) => r1 !== r2,
```
Comment (drets) [inline src/components/ArticlesPage.js:23]: Could you explain what it does?

File: src/components/ArticlesPage.js [hunk 2]
```diff
@@ -15,13 +16,35 @@ const styles = StyleSheet.create({
...
+    })
+
+    this.state = {
+      articlesDataSource: ds.cloneWithRows(props.articles.toJS()),
```
Comment (drets) [inline src/components/ArticlesPage.js:27]: and this one

File: src/components/ArticlesPage.js (hunk 1 above)
Comment (rasendubi) [reply src/components/ArticlesPage.js:23]: This is a required parameter that should tell if rows are different and need re-rendering.

File: src/components/ArticlesPage.js (hunk 2 above)
Comment (rasendubi) [reply src/components/ArticlesPage.js:27]: Check out [examples of `ListView`](http://facebook.github.io/react-native/releases/0.42/docs/listview.html#listview). Ask if something is still unclear.

**Pull request merged**